src/main/java/com/exadel/project/skill/service/SkillService.java

```diff
@@ -0,0 +1,32 @@
+package com.exadel.project.skill.service;
+
+import com.exadel.project.common.exception.EntityNotFoundException;
+import com.exadel.project.skill.entity.Skill;
+import com.exadel.project.skill.repository.SkillRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+
+@Service
+@AllArgsConstructor
+public class SkillService {
+    private SkillRepository skillRepository;
+
+    public List<Skill> getAllSkills(){
+        return skillRepository.findAll();
+    }
+
+    public List<Skill> getSkillsByIds(List<Long> ids){
+        return skillRepository.findByIdIn(ids);
+    }
+
+    public Skill getSkillById(Long id) throws EntityNotFoundException {
+        Optional<Skill> optionalSkill = skillRepository.findById(id);
+        if (optionalSkill.isPresent()){
+            return optionalSkill.get();
+        }else {
+            throw new EntityNotFoundException();
+        }
+    }
```

Comment:
Could we replace this structure with 
skillRepository.findById(id).orElseThrow(() -> new EntityNotFoundException())

Reply:
fixed